source/main/gfx/hydrax/Perlin.cpp

```diff
@@ -389,7 +389,7 @@ namespace Hydrax{namespace Noise
 			PixelBuffer->lock(Ogre::HardwareBuffer::HBL_DISCARD);
 			const Ogre::PixelBox& PixelBox = PixelBuffer->getCurrentLock();
 
-			Data = static_cast<unsigned short*>(PixelBox.data);

```

Comment:
rather change to reinterpret_cast. If you looped over uint16 before, looping over uint8 will produce wrong results.